spec/fetch.spec.js

```diff
@@ -70,7 +70,7 @@ describe('fetch/uninstall tests via npm & git', function () {
             .then(_ => uninstall('cordova-android', tmpDir, opts))
             .then(_ => expectNotToBeInstalled('cordova-android'))
 
-            .then(_ => fetchAndMatch('https://github.com/apache/cordova-browser.git', { name: 'cordova-browser' }))
+            .then(_ => fetchAndMatch('https://github.com/apache/cordova-browser.git#5.0.x', { name: 'cordova-browser' }))
```

Comment:
Why is this necessary from a technical standpoint?

Reply:
Probably because latest browser release dropped support for Node 4.